src/main/java/io/reactivex/internal/fuseable/ScalarSupplier.java

```diff
@@ -22,16 +22,16 @@
  * <p>
  * Implementors of {@link #call()} should not throw any exception.
```

Comment:
The link should now point to `get()`.

Reply:
Thanks. Posted an update with additional cleanups.